tests/spec/core/markdown-spec.js

```diff
@@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\nFoo\n===\n",
+       "\n\nFoo\n===\n",
```

Comment:
This was not valid markdown (I was surprised too!). It causes the section above to be wrapped in a p element. 

> On 5 May 2016, at 8:02 PM, Tobie Langel <notifications@github.com> wrote:
> 
> In tests/spec/core/markdown-spec.js:
> 
> > @@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
> >      var ops = {
> >        config: makeBasicConfig(),
> >        body: makeDefaultBody() +
> > -       "\nFoo\n===\n",
> > +       "\n\nFoo\n===\n",
> Why are you fixing tests, here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>